pipelines/zendesk/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pendulum
+python-dlt
+zenpy==2.0.25
```

Comment:
at least make it ^2.0.25 or even >= do not bind to exact version (if there's no reason)